runtime/vara/src/lib.rs

```diff
@@ -1048,6 +1049,23 @@ impl pallet_gear_messenger::Config for Runtime {
     type CurrentBlockNumber = Gear;
 }
 
+/// Builtin actors arranged in a tuple.
+#[cfg(not(feature = "runtime-benchmarks"))]
+pub type BuiltinActors = ();
+#[cfg(feature = "runtime-benchmarks")]
+pub type BuiltinActors = pallet_gear_builtin::benchmarking::BenchmarkingBuiltinActor<Runtime>;
+
+parameter_types! {
+    pub const BuiltinActorPalletId: PalletId = PalletId(*b"py/biact");
+}
+
+impl pallet_gear_builtin::Config for Runtime {
```

Comment:
btw couldn't find changes to gsdk generated code. are they not necessary here?

Reply:
apparently, they are not. At least the script didn't generate any new metadata. Anyway, will re-run it.